File: Pemrograman 1/MODUL 6/Bahasa C/PRAK601-2410817110002-Akhmad_Daffa_Azzikri.c
#include <stdio.h>

int main() {
    int baris, kolom;
    scanf("%d %d", &baris, &kolom);
    int matrix[baris][kolom];
    int isi[baris * kolom];

    for (int i = 0; i < baris * kolom; i++) {
        scanf("%d", &isi[i]);
    }

    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            matrix[i][j] = isi[i * kolom + j];
            printf("%d ", matrix[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: Pemrograman 1/MODUL 6/Bahasa Python/PRAK601-2410817110002-Akhmad_Daffa_Azzikri.py
baris, kolom = map(int, input().split())
isi = list(map(int, input().split()))

matrix = [isi[i * kolom:(i + 1) * kolom] for i in range(baris)]

for baris2 in matrix:
    print(' '.join(map(str, baris2)))